internal/pfcpsim/helpers.go

```diff
@@ -58,9 +53,7 @@ func DisconnectPFCPSim() error {
 		return notInit
 	}
 
-	cancelFunc()
-
-	return nil
+	return sim.TeardownAssociation()
```

Comment:
Hi @gab-arrobo 
The use of the cancelFunc is replaced by invoking `sim.TeardownAssociation()`.
TeardownAssociation calls the cancel func of the context of the pfcpSim, it will triggers the graceful shutdown procedure.